docs/configuration.md

```diff
@@ -108,3 +108,37 @@ Please see the [features section](/mockery/features/#packages-configuration) for
     | MockName | The name of the mock that will be generated. Note that this is simply the `mockname` configuration variable |
     | PackageName | The name of the package from the original interface |
     | PackagePath | The fully qualified package path of the original interface |
+
+#### Template functions
+
+!!! note
+    Templated functions are only available when using the `packages` config feature.
+
+Template functions allow you to inspect and manipulate template variables.
+
+All template functions are calling native Go functions under the hood, so signatures and return values matches the Go functions you are probably already familiar with.
+
+To learn more about the templating syntax, please [see the Go `text/template` documentation](https://pkg.go.dev/text/template){target="_blank"}
+
+* [`contains` string substr](https://pkg.go.dev/strings#Contains){target="_blank"}
```

Comment:
Hmm, maybe we should find a way to have mkdocs open all links in new tabs. That way we wouldn't have to do the `target="_blank"`. This is fine for now, it's a PR for another day.